stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/Network.java

```diff
@@ -87,8 +89,17 @@ private GetResponseBodyResponse readResponseBody(String requestId)
     return response;
   }
 
-  public AsyncPrettyPrinterRegistry getAsyncPrettyPrinterRegistry() {
-    return mNetworkPeerManager.getAsyncPrettyPrinterRegistry();
+  /**
+   * Method that allows callers to provide an
+   * {@link com.facebook.stetho.inspector.network.AsyncPrettyPrinterInitializer} that is
```

Comment:
fully qualified class name isn't needed because we have an import for it. So the javadoc can say, `Method that allows callers to provide an {@link AsyncPrettyPrinterInitialier} that is...`